### PR TITLE
create adverse drug reaction form

### PR DESCRIPTION
**Pull request by ckote**
creation of adr form for pharmacovigilance surveillance

### CODE REVIEW

File: configuration/ampathforms/Adverse_Drug_Reaction_Form.json [hunk 1]
```diff
@@ -0,0 +1,1137 @@
...
+              "questionInfo": "",
+              "id": "phoneNumberInitial",
+              "required": "false",
+              "questionOptions": {
+                "rendering": "text",
+                "attributeType": "b2c38640-2603-4629-aebd-3b54f33f1e3a",
+                "isSimpleValue": true
+              },
+              "hide": {
+                "hideWhenExpression": ""
```
Comment (makombe) [inline configuration/ampathforms/Adverse_Drug_Reaction_Form.json:1051]: Same has here

File: configuration/ampathforms/Adverse_Drug_Reaction_Form.json [hunk 2]
```diff
@@ -0,0 +1,1137 @@
...
+              "id": "phoneNumberPPB",
+              "required": "false",
+              "questionOptions": {
+                "rendering": "text",
+                "attributeType": "b2c38640-2603-4629-aebd-3b54f33f1e3a",
+                "isSimpleValue": true
+              },
+              "hide": {
+                "hideWhenExpression": ""
+              }
```
Comment (makombe) [inline configuration/ampathforms/Adverse_Drug_Reaction_Form.json:1114]: Phone number repeated

Comment (makombe) [reply configuration/ampathforms/Adverse_Drug_Reaction_Form.json:1114]: @ckote I have left a comment. What is the difference between the two phone numbers?

**Comment by ckote** [comment]
One is for the reporter while the other is for the submitter @makombe

**Review by RAJABIBRAZ — APPROVED** [review]

**Pull request merged**